src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/mvn.cpp

```diff
@@ -175,6 +175,32 @@ const auto Mvn4DStatic = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_Mvn4D_Static, MvnLayerCPUTest, Mvn4DStatic, MvnLayerCPUTest::getTestCaseName);
 
+// test cases with f32 precision, to cover non-avx512 platforms(bf16 is skipped on non-avx512 machine)
+const std::vector<ov::Shape>& inputShapesStatic_4D_CTails() {
+    static const std::vector<ov::Shape> inputShapesStatic_4D = {
+        {1, 3, 2, 2},
+        {1, 4, 5, 5},
+        {1, 7, 2, 5},
+    };
+    return inputShapesStatic_4D;
+}
+
+const auto Mvn4DStaticCTails = ::testing::Combine(
+       ::testing::Combine(
+               ::testing::ValuesIn(static_shapes_to_test_representation(inputShapesStatic_4D_CTails())),
+               ::testing::Values(ElementType::f32),
+               ::testing::ValuesIn(emptyReductionAxes()),
+               ::testing::Values(false),
+               ::testing::ValuesIn(normalizeVariance),
+               ::testing::ValuesIn(epsilon())),
+       ::testing::ValuesIn(filterCPUSpecificParams(cpuParams_4D)),
+       ::testing::ValuesIn(fusingParamsSetStaticShape),
+       ::testing::Values(ElementType::f32),
+       ::testing::Values(ElementType::f32),
+       ::testing::ValuesIn(additionalConfig()));
+
```

Comment:
`additionalConfig` contains 2 data type f32 and f16, just test f32 data type is enough if we only test SSE41,  is it right? @chenhu-wang

Reply:
yes, one precision could cover this fix, done.